src/new-client/src/App/Trades/TradesState/tableTrades.ts

```diff
@@ -162,7 +155,9 @@ const numFiltersTrueOfTrade = (
       return true
     }
 
-    const tradeNumber = trade[field as NumColField]
+    const tradeValue = trade[field as NumColField]
+    const tradeNumber =
+      typeof tradeValue === "number" ? tradeValue : parseFloat(tradeValue)
```

Comment:
Minor - I would have opted for `parseInt` to be explicit that we expect a whole number, but the outcome is the same so no need to change if you don't agree.